### ISSUE TITLE
fastlane integration

### ISSUE THREAD

**Issue by natemann** [czechboy0/XCSTutorialProject1#4]
I am having a problem integrating fastlane into a post trigger.  I have verified fastlane is installed at `/usr/local/bin/fastlane`, but we keep getting the error `fastlane: command not found`.  Any thoughts?

**Comment by czechboy0** [comment]
Right, you have to use this way on El Cap to install fastlane/cocoapods etc: https://honzadvorsky.com/articles/2015-09-14-15-00-xcode_server_hacks_3_install_cli_tools_for_xcsbuildd/
